cli/pack/pack.go

```diff
@@ -120,7 +120,7 @@ func Run(ctx *context.Ctx) error {
 		return err
 	}
 
-	if ctx.Pack.Type != DockerType || len(ctx.Pack.ImageTags) == 0 {
+	if ctx.Pack.Type != DockerType || len(ctx.Pack.ImageTags) == 0 && checkFilename(ctx) {
```

Comment:
Should it really be here?

Reply:
It saves us few operations if --filename is specified